ext/sockets/sockets.c

```diff
@@ -2089,7 +2097,15 @@ PHP_FUNCTION(socket_create_pair)
 		RETURN_THROWS();
 	}
 
-	if (type > 10) {
+	stype = type;
```

Comment:
maybe comment for this would be good - it took me a while to figure out that stype is just temporary for that check only. Or maybe rename that var to be clear from the name.